components/servo/lib.rs

```diff
@@ -1016,23 +1016,32 @@ pub fn run_content_process(token: String) {
         create_sandbox();
     }
 
-    let background_hang_monitor_register =
-        unprivileged_content.register_with_background_hang_monitor();
-
     // send the required channels to the service worker manager
     let sw_senders = unprivileged_content.swmanager_senders();
     let _js_engine_setup = script::init();
     script::init_service_workers(sw_senders);
 
     media_platform::init();
-
-    unprivileged_content.start_all::<script_layout_interface::message::Msg,
-                                     layout_thread::LayoutThread,
-                                     script::script_thread::ScriptThread>(
-                                         true,
-                                         background_hang_monitor_register,
-                                         None,
-                                     );
+    if opts::get().background_hang_monitor {
```

Comment:
It might be clearer to do something like:

```rust
let background_hang_monitor_register = if opts::get().background_hang_monitor {
     unprivileged_content.register_with_background_hang_monitor()
} else {
     None
};
```

Reply:
Indeed!